docs/release-notes.md

```diff
@@ -1,115 +1,211 @@
 
-**AMT Clock Sync Maintenance Task:** Intel&reg; AMT requires periodic maintenance to keep the real-time clock synced so that accurate certificate checking can be performed.  In this release, we have added this capability to both RPS and RPC.  A new "maintenance" command has been added to our RPC-Go application that will call back to RPS and perform the clock sync task.
+### Noteworthy Features and Changes
+**Major Refactoring & WSMAN-MESSAGES Library:** Starting at the end of 2021, the team took on a major task of refactoring some of the deeper parts of our code.  This part of the code was fundamental in how the rest of the Open AMT Cloud Toolkit talked to Intel&reg; AMT devices. Much of this code was inherited from another open source project also developed at Intel.  While this inherited code is well validated and stable, it is also very difficult to use automated testing tools to validate when changes are required.  For those who care about code coverage indicators when assessing if they should use open source libraries, this made the Open AMT Cloud Toolkit an unattractive solution.  With these factors in mind, we created [WSMAN-MESSAGES](https://github.com/open-amt-cloud-toolkit/wsman-messages) a portable and testable library that handles the creation of wsman based messages for talking to Intel&reg; AMT devices.  Wsman-Messages is used by both MPS and RPS is available on NPM as [@open-amt-cloud-toolkit/wsman-message](https://www.npmjs.com/package/@open-amt-cloud-toolkit/wsman-messages).  Developers can independently make use of this library if they want to create and send messages to Intel&reg; AMT devices.  The net result from this refactoring effort is that code coverage in MPS went from 24% to 91% and RPS went from 29% to 70%.  This is a huge leap in reducing the overall technical debt of the project and should provide our customers with more confidence adopting the Open AMT Cloud Toolkit.  As we move forward, we'll continue to work towards our goal of 80% code coverage across all repositories and these additional improvements will happen through the course of our normal release cadence.
 
-**Hostname Override:** In response to customer feedback, we have added the ability to override the hostname using a -h command in RPC-Go.  Instead of sending up the HostOS name, RPC will now send up the host name specified.  This is particularly helpful when running RPC from a container.
+**Device Migration:** Through working with our customers, we've had several requests for a way to move an Intel&reg; AMT device from one management console to the the Open AMT Cloud Toolkit.  With 2.2, you can perform this migration with RPC by using the activate command and provide the profile you wish to use to activate the device.  By providing the AMT credential at the prompt, RPS will use this credential to add the device to Open AMT Cloud Toolkit and then apply the configuration profile, completing the migration without needing to deactivate the device.
```

Comment:
**Device Migration:** Several customers have requested a method to migrate an Intel&reg; AMT device from a management console to the Open AMT Cloud Toolkit. Now with 2.2, perform this migration with RPC without deactivating and reactivating the device. Use the activate command and provide the desired profile. The prompt will ask for the AMT credential. RPS will use this credential to add the device to Open AMT Cloud Toolkit and apply the configuration profile.